e2e/testcases/apiservice.bats

```diff
@@ -13,17 +13,32 @@ YAML_DIR=${BATS_TEST_DIRNAME}/../testdata/apiservice
 test_setup() {
   setup::git::initialize
   setup::git::commit_minimal_repo_contents
+  wait::for -t 120 -- nomos::repo_synced
```

Comment:
In the end of `setup::git::commit_minimal_repo_contents`, it waits for the repo to be synced, so the additional wait is unnecessary.

Reply:
Done! Didn't notice that.